docs/credentials-provision.md

```diff
@@ -0,0 +1,17 @@
+# Credentials provisioning tool
+
+To assist with the creating and maintenance of cloud credentials from outside of the cluster (necessary when CCO is put in "Manual" mode), the credentials-provision tool provides tooling.
+
+## Global flags
+
+By default, the tool will output to the directory the command(s) were run in. To specify a directory, use the `--output-dir` flag.
+
+## Creating RSA keys
+
+To generate keys for use when setting up the cluster's OpenID Connect provider, run
+
+```bash
+$ credentials-provision create key-pair
```

Comment:
```suggestion
credentials-provisioning create key-pair
```
As written in the code

Reply:
hmm..maybe 'credentials-provision' is a slightly better name...

Reply:
+1 for credentials-provision

Reply:
> Ideas:
> 
> * cco
> * ccoctl
> * ccoadm

I will vote for ccoctl here